paint-timing/fcp-only/fcp-gradient.html

```diff
@@ -7,26 +7,22 @@
         height: 100px;
         background-image: linear-gradient(to bottom, orange, blue);
     }
-
-    .contentful {
-        background-image: url(../resources/circles.png);
-    }
-
-    #log {
-        display: none;
-    }
 </style>
 </head>
 <body>
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script src="../resources/utils.js"></script>
 <div id="main"></div>
-</body>
-
-<footer>
 <script>
-    test_fcp("First contentful paint doesn't fire for gradients.")
+  promise_test(async t => {
+      assert_precondition(window.PerformancePaintTiming, "Paint Timing isn't supported.");
+      const main = document.getElementById('main');
+      await new Promise(r => window.addEventListener('load', r));
+      await waitForAnimationFrames(numFramesWaiting);
+      assert_false(didReceiveFCP(), 'FCP marked too early.');
```

Comment:
Can this just be assertNoFCP?

Reply:
No, because assertNoFCP taints the test by adding the log too early... I'm using this to avoid having to touch the #log div.

Reply:
Refactored.